src/hw7_Map_Set/coffee_order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,53 @@
+package hw7_Map_Set.coffee_order;
+
+import java.util.Deque;
+import java.util.LinkedList;
+
+public class CoffeeOrderBoard {
+    final Deque<Order> orders;
+    private static int orderCounter = 0;
+
+    public CoffeeOrderBoard() {
+        orders = new LinkedList<>();
+    }
+
+    public void add(Order order){
```

Comment:
Here we can retrieve only customer name and create Order object inside add method